### PR TITLE
feat: add mobile flag to disable in prod

### PR DESCRIPTION
**Pull request by tuchfarber**
**Description:**
Describe in a couple of sentences what this PR adds

**Merge checklist:**
- [x] Bump version
- [x] Add to changelog
- [x] Update docs (not only docstrings)

**Post merge:**
(Will eventually be handled via automation)
- [ ] Release with new tag matching version

**Author concerns:** List any concerns about this PR - inelegant
solutions, hacks, quick-and-dirty implementations, concerns about
migrations, etc.

### CODE REVIEW

File: notices/settings/common.py [hunk 1]
```diff
@@ -7,6 +7,7 @@ def plugin_settings(settings):
     settings.FEATURES["NOTICES_DEFAULT_REDIRECT_URL"] = "http://www.example.com"
     settings.FEATURES["NOTICES_FALLBACK_LANGUAGE"] = "en"
     settings.FEATURES["NOTICES_SEGMENT_KEY"] = None
-    settings.FEATURES["NOTICES_SNOOZE_HOURS"] = None
-    settings.FEATURES["NOTICES_SNOOZE_COUNT_LIMIT"] = None
-    settings.FEATURES["NOTICES_MAX_SNOOZE_DAYS"] = None
+    settings.FEATURES["NOTICES_SNOOZE_HOURS"] = 24
+    settings.FEATURES["NOTICES_SNOOZE_COUNT_LIMIT"] = 3
+    settings.FEATURES["NOTICES_MAX_SNOOZE_DAYS"] = 30
```
Comment (oliviaruizknott) [inline notices/settings/common.py:12]: Did you mean to change these other ones from `None`s to these values?

**Review by oliviaruizknott — APPROVED** [review]
Looks great! One question about changing defaults but feel free to merge if that’s correct or once you change it.

Oh, sorry about the Changelog conflicts. 😅

File: notices/settings/common.py (hunk 1 above)
Comment (tuchfarber) [reply notices/settings/common.py]: Eek! Good catch. These were from testing. Didn't see the change. Fixed!

**Comment by edx-deployment** [comment]
A PR has been created in edx-internal to release this new version. Please review it here: https://github.com/edx/edx-internal/pull/5707

**Pull request merged**